scripts/fetch-tags.mjs

```diff
@@ -0,0 +1,46 @@
+import { execSync } from 'child_process'
+import execa from 'execa'
+;(async () => {
+  let commitId = process.argv[process.argv.length - 1]
+
+  if (commitId.endsWith('fetch-tags.mjs')) {
+    commitId = ''
+  }
+
+  // <hash> (<tag>) <message>
+  // parse only the last string which should be version if
+  // it's a publish commit
+  const commitMsg = execSync(
+    `git log --oneline -n 1${commitId ? ` ${commitId}` : ''}`
```

Comment:
It might be better to use something like `git log --pretty=format:'%h'` instead of `--oneline` because the format might change between versions of git

https://git-scm.com/docs/pretty-formats

Reply:
Good point, we only need the body so updated to format and only log that